pipelines/fbc-builder/patch.yaml

```diff
@@ -24,6 +24,8 @@
     value: $(params.path-context)
   - name: DOCKER_AUTH
     value: "$(tasks.appstudio-configure-build.results.docker-auth)"
+  - name: HERMETIC
+    value: "true"
```

Comment:
Just a heads-up that this will make the prefetch task run and [print](https://github.com/redhat-appstudio/build-definitions/blob/main/task/prefetch-dependencies/0.1/prefetch-dependencies.yaml#L27): 

```
Build will be executed with network isolation, but no content was configured to be prefetched.
This will likely result in a build failure.
```

Maybe we should get rid of the last sentence to avoid user confusion.